packages/database-editor/src/components/ImportWizard/TableDropdown.tsx

```diff
@@ -0,0 +1,42 @@
+import type { DatabaseTable } from '@axonivy/database-editor-protocol';
+import { Button, DropdownMenu, DropdownMenuCheckboxItem, DropdownMenuContent, DropdownMenuTrigger, IvyIcon } from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { useTranslation } from 'react-i18next';
+import './TableDropdown.css';
+
+export const TableDropdown = ({
```

Comment:
I think I would streamline this name with the known name -> TableSelect or TableMultiSelect